src/OptionsOverlay.tsx

```diff
@@ -83,6 +85,7 @@ export const OptionsOverlay = <T extends Options>(
                 })),
               ];
               onAllMenuOptionsUpdate(updatedOptions as T);
+              onOptionClick();
```

Comment:
What do you think of returning the option here? Might be useful right?

Reply:
Sound good to me. Will do that.